.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
 services:
   - postgresql
 before_script:
-  - psql -c 'create database jubilee_test;' -U postgres
+  - psql -c 'create database jubilee_ test;' -U postgres
```

Comment:
Why are we changing the database name here?

Reply:
That was an oversight. Have been corrected